dev/io.openliberty.netty.internal.impl/src/io/openliberty/netty/internal/impl/NettyFrameworkImpl.java

```diff
@@ -304,37 +300,53 @@ public ChannelFuture start(BootstrapExtended bootstrap, String inetHost, int ine
         return UDPUtils.start(this, bootstrap, inetHost, inetPort, bindListener);
     }
 
+    @Override
+    public ChannelFuture startOutbound(BootstrapExtended bootstrap, String inetHost, int inetPort,
+            ChannelFutureListener bindListener) throws NettyException {
+        if (bootstrap.getConfiguration() instanceof TCPConfigurationImpl) {
+            return TCPUtils.startOutbound(this, bootstrap, inetHost, inetPort, bindListener);
+        } else {
+            return UDPUtils.startOutbound(this, bootstrap, inetHost, inetPort, bindListener);
+        }
+    }
+
     @Override
     public ChannelFuture stop(Channel channel) {
-        if (isActive && channel.isOpen()) {
-            ChannelFuture closeFuture = channel.close();
-            activeChannels.remove(closeFuture.channel());
-            closeFuture.addListener(future -> {
-                if (!future.isSuccess()) {
-                    if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
-                        String channelName = channel.attr(ConfigConstants.NameKey).get();
-                        Tr.debug(tc,
-                                "channel " + channelName + " failed to stop due to: " + future.cause().getMessage());
+        if (activeChannels.contains(channel)) {
+            activeChannels.remove(channel);
```

Comment:
swaps the order of stop and remove from active?

Reply:
also now if serverStopping has been called which set isActive to false a call to this method
will remove the channel from the active set but will not close it.

Reply:
Yeah this seems a bit weird I'll need to revisit this and see what the logic was to change this order

Reply:
I've gone through it but I'm not sure if this actually messes anything up due to the fact that activeChannels is thread safe but there could be a pathway where this is possible. I created https://github.com/OpenLiberty/open-liberty/issues/23441 to follow up on this and investigate further.

Reply:
OK (I think - worth talking through and raising a followup issue is fair enough.)